src/cascadia/TerminalSettingsModel/CascadiaSettings.cpp

```diff
@@ -326,6 +326,9 @@ Model::Profile CascadiaSettings::DuplicateProfile(const Model::Profile& source)
         DUPLICATE_SETTING_MACRO_SUB(appearance, target, SelectionBackground);
         DUPLICATE_SETTING_MACRO_SUB(appearance, target, CursorColor);
         DUPLICATE_SETTING_MACRO_SUB(appearance, target, Opacity);
+        DUPLICATE_SETTING_MACRO_SUB(appearance, target, DarkColorSchemeName);
+        DUPLICATE_SETTING_MACRO_SUB(appearance, target, LightColorSchemeName);
+        DUPLICATE_SETTING_MACRO_SUB(appearance, target, ColorSchemeName);

```

Comment:
`ColorSchemeName` was already added in the macro above because it's a part of `MTSM_APPEARANCE_SETTINGS` in MTSMSettings.h.

Since that's the case, maybe we should be replacing it with `DarkColorSchemeName` and `LightColorSchemeName` there too?